src/mlpack/methods/reinforcement_learning/noise/gaussian.hpp

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file methods/reinforcement_learning/noise/gaussian.hpp
+ * @author Tarek Elsayed
+ *
+ * This file is the implementation of GaussianNoise class.
+ * Gaussian Noise is statistical noise with a normal distribution.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_RL_NOISE_GAUSSIAN_HPP
+#define MLPACK_METHODS_RL_NOISE_GAUSSIAN_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+class GaussianNoise
+{
+ public:
+  /**
+   * @param size The size of the noise vector.
+   * @param mu The mean of the noise.
+   * @param sigma The standard deviation of the noise.
+   */
+  GaussianNoise(int size,
+          double mu = 0.0,
+          double sigma = 1.0) :
```

Comment:
Do you mind to align this with the `size` parameter, also, should we use `const` for every parameter?